sensor_array/helper/generate_comps_for_simulations.py

```diff
@@ -1,5 +1,13 @@
 import numpy as np
 import csv
+import yaml
+
+
+def yaml_loader(filepath):
+    with open(filepath, 'r') as yaml_file:
+        data = yaml.load(yaml_file)
+    return data
+
```

Comment:
Your different scripts can share a helper file that has yaml_loader in it, maybe a few other shared things. They don't need to be completely distinct.